lib/MixedModels/LMM.rb

```diff
@@ -149,6 +155,8 @@ def initialize(x:, y:, zt:, lambdat:, weights: nil, offset: 0.0, reml: true,
   #                      are on the right. 
   # * +data+           - a Daru::DataFrame object, containing the response, fixed and random 
   #                      effects, as well as the grouping variables
+  #Maybe the following parameters which appear in all three of these functions should
+  #be encapsulated in a FitOptions class. On the hand, maybe not.
```

Comment:
My plan is to implement alternative optimization algorithms, or somehow interface a well-tested minimization library written in C (like NLopt). Then I would do something like
`..., minimizer: :nelder_mead, **minimizer_options)`,
where `**minimizer_options` would collect `start_point:`, `epsilon:`, `max_iterations:` or other optimization options in a Hash (if I remember this syntax correctly).
But maybe an encapsulating class would be better.